airflow/Chart.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
   repository: file://../postgresql
-  version: 0.13.1-p3
+  version: 0.13.1-p4
```

Comment:
semver thoughts would apply here too

this is becoming more important in the context of release management strategy for our team